### DISCUSSION TITLE
Prisma not working in cpanal share hosting

### DISCUSSION THREAD

**Discussion by mahbubhswe** [prisma/prisma#16368] [Q&A]
Can i use prisma in cpanel? I have tried but it give me an error like internal server error but without prisma work perfectly..........and in vercel server work perfectly only problem in cpanel

I have used Nextjs+prisma+mysql

#plz_help_me

**Comment by nurul3101** [comment]
Hello @mahbubhswe 👋 

Yes, you can deploy a node app using Prisma to Cpanel. Here's a related GitHub Discussion: [#3163](https://github.com/prisma/prisma/discussions/3163)
Can you please elaborate on what error are you getting?

**Reply by mahbubhswe** [reply]
yes..... I have deployed several time by removing and adding prisma, without prisma it works......my api work perfectly if i return a static value like array of object as a responce. When i try to connect database it give me an error internal server error.

Plz...check my deployment>>>
frontend: http://test.me.allpostbd.com/ (work)
api: http://test.me.allpostbd.com/api/services/getServices (not work)

**Reply by nurul3101** [reply]
Do you get any error stack trace along with the `Internal Server error` message?
Also it seems the deployment is moved to some other site.

**Reply by pantharshit00** [reply]
👋 Hello @mahbubhswe,  did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions!
 
 Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Reply by peter-mwau** [reply]
Hello, facing the same problem with nodejs, prisma & mysql. Did you get a solution for this @mahbubhswe @nurul3101

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
